.travis.yml

```diff
@@ -59,6 +59,10 @@ jobs:
       env: TARGETS="-C libbeat crosscompile"
       go: $GO_VERSION
       stage: test
+    - os: linux
```

Comment:
We must make sure to add these tests also to Jenkins

Reply:
@urso Can you open a follow up issue to add it to Jenkins?

Reply:
See #5821